appinventor/components/src/com/google/appinventor/components/runtime/ContextMenu.java

```diff
@@ -0,0 +1,107 @@
+// -*- mode: java; c-basic-offset: 2; -*-
+// Copyright 2020 MIT, All rights reserved
+// Released under the Apache License, Version 2.0
+// http://www.apache.org/licenses/LICENSE-2.0
+
+package com.google.appinventor.components.runtime;
+
+import android.app.Activity;
+import android.view.View;
+import com.google.appinventor.components.annotations.DesignerComponent;
+import com.google.appinventor.components.annotations.SimpleObject;
+import com.google.appinventor.components.common.ComponentCategory;
+import com.google.appinventor.components.common.YaVersion;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Container for ContextMenuItems
```

Comment:
This Javadoc will be used for the user-facing documentation, so you should provide a longer description of how the component works, what it's used for, how do you associate it with a component, etc.